chainbridge/src/lib.rs

```diff
@@ -149,11 +184,21 @@ decl_storage! {
 
 decl_module! {
     pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+        type Error = Error<T>;
+
         const ChainIdentity: ChainId = T::ChainId::get();
 
         fn deposit_event() = default;
 
-        /// Sets the vote threshold for proposals
+        /// Sets the vote threshold for proposals.
+        ///
+        /// This threshold is used to determine how many votes are required
+        /// before a proposal is executed.
+        ///
+        /// # <weight>
+        /// - O(1) lookup and insert
+        /// # </weight>
+        #[weight = SimpleDispatchInfo::FixedNormal(500_000)]
```

Comment:
Not really sure how to compare these weights with. But looks to me that we should make this more expensive, although is coming from democracy in our case. Any ideas @philipstanislaus ?

Reply:
Weights look fine for me. A balances transfer is 1_000_000, just for reference.